From d6df880387fe45229c5698cfcdae671f114d6438 Mon Sep 17 00:00:00 2001
From: "xiaoya.lin" <xiaoya.lin@amlogic.com>
Date: Wed, 19 Jul 2023 14:32:37 +0800
Subject: [PATCH] 265_encoder: CF1 modify cma buff size [1/1]

PD#SWPL-132055

Problem:
Need modify cma buff size

Solution:
Modify cma buff size

Verify:
Local

Signed-off-by: xiaoya.lin <xiaoya.lin@amlogic.com>
Change-Id: I3eeab53b669d6282cc2f2215e92b79f3e6c30c6c
---
 drivers/frame_sink/encoder/h265/vpu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/frame_sink/encoder/h265/vpu.c b/drivers/frame_sink/encoder/h265/vpu.c
index 642886c7c..198c1858b 100644
--- a/drivers/frame_sink/encoder/h265/vpu.c
+++ b/drivers/frame_sink/encoder/h265/vpu.c
@@ -78,7 +78,7 @@
 
 #define MHz (1000000)
 
-#define VPU_INIT_VIDEO_MEMORY_SIZE_IN_BYTE (256 * SZ_1M)
+#define VPU_INIT_VIDEO_MEMORY_SIZE_IN_BYTE (100 * SZ_1M)
 
 #define LOG_ALL 0
 #define LOG_INFO 1
-- 
2.25.1

